src/cli/cli.cpp

```diff
@@ -1240,24 +1240,18 @@ otError Interpreter::ProcessBackboneRouterLocal(Arg aArgs[])
 }
 #endif // OPENTHREAD_FTD && OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE
 
+/**
+ * @cli domainname
```

Comment:
For my future reference, you would prefer to have these (no argument instances) before the `template` line even when there are other arguments?  We have been putting them just before the start of the logic for no arg, if there's another logic block for arguments.  Otherwise we put them before the `template` line like you have here.

Reply:
Yes. This sounds good to me.